core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/DefaultViewMetadata.java

```diff
@@ -81,8 +81,8 @@ public CqlIdentifier getName() {
 
   @NonNull
   @Override
-  public UUID getId() {
-    return id;
+  public Optional<UUID> getId() {
+    return Optional.of(id);
```

Comment:
If `id` can be null than this should rather be `Optional.ofNullable(id)`. (But I'm not sure if views can be virtual, can they?)

Reply:
I don't believe that views can be virtual at this time, just keyspaces and tables